Mathlib/Data/LazyList/Basic.lean

```diff
@@ -11,11 +11,10 @@ import Mathlib.Lean.Thunk
 /-!
 ## Definitions on lazy lists
 
-This file contains various definitions and proofs on lazy lists.
-
-TODO: This file will soon be deprecated.
+This file is entirely deprecated, and contains various definitions and proofs on lazy lists.
```

Comment:
I was wondering if we should add a pointer to `MLList` here, but it's probably fine since `LazyList` already says so in the deprecation warning.